src/list.scss

```diff
@@ -387,4 +403,138 @@ $block: #{$fd-namespace}-list;
       }
     }
   }
+
+  // Standard List with Byline
+  &--byline {
+    .#{$block}__item {
+      width: 100%;
+      height: $fd-list-byline-item-height;
+      padding: $fd-list-byline-item-padding-x $fd-list-byline-item-padding-y;
+
+      @include fd-active() {
+        .#{$block}__thumbnail,
+        .#{$block}__byline,
+        .#{$block}__byline-left,
+        .#{$block}__byline-right {
+          color: var(--sapList_Active_TextColor);
+        }
+
+        @each $set-name, $color-set in $semantic-color {
+          &--#{$set-name} {
+            color: var(--sapList_Active_TextColor);
+          }
+        }
+      }
+    }
+
+    .#{$block}__thumbnail {
+      @include fd-reset();
+      @include fd-flex-center();
+
+      width: $fd-list-byline-thumbnail-dimensions;
+      height: $fd-list-byline-thumbnail-dimensions;
+      min-width: $fd-list-byline-thumbnail-dimensions;
+      max-width: $fd-list-byline-thumbnail-dimensions;
+      min-height: $fd-list-byline-thumbnail-dimensions;
+      max-height: $fd-list-byline-thumbnail-dimensions;
```

Comment:
If you wonder why I have to use min-width, width, max-width and set them to the same value it's because we need to handle the case when an image (fd-image) is used and for this reason it needs to overwrite the min-width/max-width, min-height/max-height properties of the image. We are currently migrating Image to Avatar so this might be refactored later to only use width and height.

Reply:
It also helps IE11 to interpret the code. Sometimes it's even mandatory to keep fixed width.